File: Ej2_Java/src/main/java/org/example/Videojuego.java
package org.example;

public class Videojuego {
    String titulo;
    String plataforma;
    String horasJugadas;

    public Videojuego(String titulo, String plataforma, String horasJugadas) {
        this.titulo = titulo;
        this.plataforma = plataforma;
        this.horasJugadas = horasJugadas;
    }

    public void mostrarInfo(){
        System.out.println("Titulo: " + titulo + ", Plataforma: " + plataforma + ", Horas Jugadas: " + horasJugadas);
    }

}

File: Ej2_Kotlin/src/main/kotlin/Videojuego.kt
package org.example

data class Videojuego(
    val titulo: String,
    val plataforma: String,
    val horasJugadas: Int
){
    override fun toString(): String {
        return "Titulo: $titulo, Plataforma: $plataforma, Horas de juego: $horasJugadas"
    }
}